sips/sip-010/sip-010-fungible-token-standard.md

```diff
@@ -0,0 +1,172 @@
+# Preamble
+
+Sip Number: 010
+
+Title: Standard Trait Definition for Non-Fungibe Tokens
+
+Author: Hank Stoever (hstove@gmail.com)
```

Comment:
We also giving @psq credit?

Reply:
I definitely want to - I [asked him for an email previously](https://github.com/stacksgov/sips/pull/5#discussion_r565068292) but haven't heard back on that thread.

Reply:
Gotcha.  We can always add it in later, after ratification.

Reply:
@hstove well you can always use the one in his git commits `<psq@nanorails.com>`

Reply:
@hstove not sure I ever got that request, but `psq@nanorails.com` is fine to use here

Reply:
Added Pascal as a contributor :+1: